test/test_rake_task_arguments.rb

```diff
@@ -124,4 +131,10 @@ def test_extra_args_with_less_than_named_arguments
     assert_equal [], ta.extras
   end
 
+  private
+
+  def older_ruby?
```

Comment:
I removed this at https://github.com/ruby/rake/commit/95f212c5171dbc366e32ec56d9af61492728125d